lib/datasource/galaxy-collection/index.ts

```diff
@@ -0,0 +1,156 @@
+import { logger } from '../../logger';
+import { ExternalHostError } from '../../types/errors/external-host-error';
+import * as packageCache from '../../util/cache/package';
+import { Http } from '../../util/http';
+import type { GetReleasesConfig, Release, ReleaseResult } from '../types';
+
+export const id = 'galaxy-collection';
+export const defaultRegistryUrls = ['https://galaxy.ansible.com/'];
+export const customRegistrySupport = false;
+
+const http = new Http(id);
+
+interface BaseProjectResult {
+  versions_url: string;
+  deprecated: boolean;
+  latest_version: {
+    version: string;
+  };
+}
+
+interface VersionsProjectResult {
+  results: Versions[];
+}
+
+interface VersionsDetailResult {
+  download_url: string;
+  artifact: {
+    filename: string;
+    size: bigint;
+    sha256: string;
+  };
+  metadata: {
+    homepage: string;
+    tags: Record<string, string>;
+    dependencies: Record<string, string>;
+    repository: string;
+  };
+}
+
+interface Versions {
+  version: string;
+  href: string;
+}
```

Comment:
move interfaces to new `./types.ts` file

Reply:
I have set it up like that at first, but then the linter fails.    
![image](https://user-images.githubusercontent.com/17493763/113857862-5404db80-97a3-11eb-849a-cc019578d341.png)

Reply:
You need to export them